functions/sg/main.py

```diff
@@ -0,0 +1,107 @@
+import os
+import re
+import requests
+
+WFL_URL = os.environ.get('WFL_URL')
+CROMWELL_URL = os.environ.get('CROMWELL_URL')
+WORKLOAD_PROJECT = os.environ.get('WORKLOAD_PROJECT')
+OUTPUT_BUCKET = os.environ.get('OUTPUT_BUCKET')
+PATH_PATTERN = os.environ.get('PATH_PATTERN')
```

Comment:
Where does the `PATH_PATTERN` get set? I don't see it passed in as an environment variable

Reply:
It isn't currently set. At one point I think the plan was to put this function on `gs://broad-gotc-prod-storage`, and this pattern could be used so we wouldn't trigger on every bam. I suppose our plans have changed enough that this functionality probably isn't necessary, I'll go ahead and remove it.

Reply:
Ok, thanks for clarifying!